README.md

```diff
@@ -1,8 +1,8 @@
 [ ![Codeship Status for databraid-dashboard/transit-api](https://app.codeship.com/projects/459d8c80-5ead-0135-6b43-16bdadfc7181/status?branch=master)](https://app.codeship.com/projects/238758)
 
-# Databraid API Starter Repo Without Postgres
+# Description
 
-Use this repo as a starter repo for developing API servers without Postgres
+An API that uses Express to give directions from point A to point B and public transit alerts for the San Francisco Bay Area. This API uses the Google Directions API to retrieve the route and travel information between an origin and a destination by public transit. This API uses uses the 511.org API to retrieve alerts information for the bay area and parse the alerts for information on what bus lines are affected.
```

Comment:
@Sanderson239 I like this, it is concise and to the point. I think you should modify the first sentence to make mention of the fact that the API serves information allowing clients to provide real time transit information about the best transit option for a given route.